tests/test_envlist.py

```diff
@@ -351,15 +351,15 @@ def test_travis_factors_py35(self, tmpdir, monkeypatch):
         """Test python factor given in the new travis section."""
```

Comment:
The name of this test needs to be updated.

Reply:
Done